requirements.json

```diff
@@ -42,9 +42,9 @@
   },
   "virtualbox.exe": {
     "bundle": "no",
-    "version": "5.0.8",
-    "revision": "103449",
-    "url": "http://developers.redhat.com/redirect/to/virtualbox-5.0.8.download",
-    "sha256sum": "767af01ea46626bf4cf4d607c18c863501b959b41115eabddef5604d522f6280"
+    "version": "5.0.16",
+    "revision": "105871",
+    "url": "http://download.virtualbox.org/virtualbox/5.0.16/VirtualBox-5.0.16-105871-Win.exe",
```

Comment:
we should not use direct URL. We should use redirects from developers.redhat.com. See https://issues.jboss.org/browse/ORG-3019